pulser-core/pulser/backend/operator.py

```diff
@@ -140,3 +151,16 @@ def from_operator_repr(
             The constructed operator.
         """
         pass
+
+    def _to_abstract_repr(self) -> dict[str, Any]:
+        if not self._eigenstates or not self._n_qudits or not self._operations:
```

Comment:
I would choose the safer `self._eigenstates is None` etc given that any python value implicitely converts to bool